components/TextToggle/TextToggle.tsx

```diff
@@ -38,6 +34,7 @@ const TextToggle: React.FC<TextToggleProps<any>> = ({
 const ToggleButton = styled(Body)<{ $active: boolean }>`
 	cursor: pointer;
 	text-transform: capitalize;
+	color: ${(props) => props.theme.colors.selectedTheme.newTheme.text.quaternary};
```

Comment:
I think we can rename it `text.disabled` or something more relevant.

Reply:
Good idea. Updated.